Gaijinizer patch/Map344.txt

```diff
@@ -53,55 +53,54 @@ Go inside?#GT
 
 なんだか嫌な予感がして、来てみたよ
 > CONTEXT: Map344/events/13/pages/1/0/Dialogue/hyakki
-...This time, I thought it wouldn’t be
-the turn, but...
-
-Somehow I had a bad feeling and tried to come#GT
+...I didn't think I'd be making an appearance
+this time. But I had a really bad feeling and
+thought I'd better come.#Ron
 > END STRING
 
 > BEGIN STRING
 特に真新しいもんはないけど、見ていくかい？
 
 ………必要になるかもしれないだろ
 > CONTEXT: Map344/events/13/pages/1/5/Dialogue/hyakki
-There is nothing particularly brand new,
-but do you want to see it?
+I don't have anything particularly new in stock,
+but why not have a look?
 
-......You may need it.#GT
+...You might find something you need.#Ron
 > END STRING
 
 > BEGIN STRING
 \\C[10]そういえば……\\N[1]
 
 メールの確認の仕方、覚えてるかい？
 > CONTEXT: Map344/events/13/pages/1/53/Dialogue/hyakki
-\\C[10] By the way, \\N[1]...
+\\C[10]By the way, \\N[1]...
 
-Do you remember how to check your email?#Triketos
+Do you remember how to check your emails?#Triketos
 > END STRING
 
 > BEGIN STRING
 …………？
 
 メール？何の？
 > CONTEXT: Map344/events/13/pages/1/57/Dialogue/kazuya
-…………?
+...?
 
-Email? What?#GT
+Emails? What emails?#Ron
 > END STRING
```

Comment:
These should all be the non plural, `email`.

(You don't go check the mails, you check the mail.)

Reply:
lol Might be a regional thing. I've frequently heard "check your emails" as in "look through your received emails to find the one I'm talking about" as opposed to "check your email" as in "check your email account for the email I'm talking about".

Changed anyway, though.